src/Input/InputWrapper.tsx

```diff
@@ -0,0 +1,121 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import React from "react";
+import * as typography from "../typography";
+import * as colors from "../colors";
+const IconError = require(`../../icons/IconError.tsx`).IconError;
+const IconInfo = require(`../../icons/IconInfo.tsx`).IconInfo;
+
+interface Props {
+  /**
+   * Visible description
+   */
+  description?: React.ReactNode;
+
+  /**
+   * Defaults to standard
+   */
+  size: "small" | "standard" | "large";
+
+  /**
+   * Visible title
+   */
+  label?: React.ReactNode;
+
+  /**
+   * If this exists, it will be displayed as an error message
+   */
+  error?: React.ReactNode;
+
+  /**
+   * If this exists, it will be displayed as an info message
+   */
+  info?: React.ReactNode;
+
+  /**
+   * an icon to have at the right of the input box, a search for example
+   */
+  icon?: React.ReactNode;
+
+  /**
+   * generally a Stepper or an Input component
+   */
+  children: React.ReactNode;
+}
+
+export function InputWrapper({
+  label,
+  description,
+  size = "standard",
+  error,
+  info,
+  icon,
+  children,
+}: Props): ReturnType<React.FC> {
+  return (
+    <React.Fragment>
+      <label
```

Comment:
We might want to have `<label/>` tag wrap the `{label}`, `{description}`, and `{children}` (not the error or info messages though), it would automatically associate the text we're using to describe the input with the input box for accessibility ([more info](https://stackoverflow.com/questions/774054/should-i-put-input-elements-inside-a-label-element)). It'd also give focus to the input when clicking on the title, which is imo desirable  

Alternatively, we could use `aria-label` or `aria-labelledby`, but those are more of a hassle (need to associate them by id, generating unique ids for association is a bit more of a hassle than associating via nesting hierarchy)